pkgs/by-name/ha/halloy/package.nix

```
@@ -43,6 +44,7 @@ rustPlatform.buildRustPackage rec {
 
   buildInputs =
     [
+      alsa-lib.dev
```

.dev should be used by default here. See: https://github.com/NixOS/nixpkgs/blob/c59414d3bb6630300418fac44e201ca638863aaf/pkgs/stdenv/generic/make-derivation.nix#L304-L331

```suggestion
      alsa-lib
```

---

@Scrumplex thank you, I have made the change and ammended the commit.